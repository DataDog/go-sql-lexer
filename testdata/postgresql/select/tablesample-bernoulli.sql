SELECT * FROM users TABLESAMPLE BERNOULLI (10);
