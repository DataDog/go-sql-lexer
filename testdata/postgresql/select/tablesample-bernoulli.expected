[
    {
        "expected": "SELECT * FROM users TABLESAMPLE BERNOULLI ( ? )",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]